test/extended/image_ecosystem/helper.go

```diff
@@ -54,3 +55,32 @@ func CheckPageContains(oc *exutil.CLI, endpoint, path, contents string) (bool, e
 	}
 	return success, nil
 }
+
+// CheckPageRegexp makes a http request for an example application and checks
+// that the result satisfies a given regexp; it will also return the first character after
+// the splitter string, for possible comparisons
+func CheckPageRegexp(oc *exutil.CLI, endpoint, path, regex, splitter string) (bool, string, error) {
+	address, err := exutil.GetEndpointAddress(oc, endpoint)
+	if err != nil {
+		return false, "", err
+	}
+
+	response, err := exutil.FetchURL(fmt.Sprintf("http://%s/%s", address, path), 3*time.Minute)
+	if err != nil {
+		return false, "", err
+	}
+
+	val := ""
+	r, _ := regexp.Compile(regex)
+	success := r.MatchString(response)
```

Comment:
seems like this is asking for grouping/submatching:
https://golang.org/pkg/regexp/#Regexp.FindStringSubmatch

you can just capture the count as a group in your regex and then return it (if matched).